src/components/input/InputRoot.tsx

```diff
@@ -30,7 +30,8 @@ const InputRootNext = function InputRoot({
 
   ...htmlAttributes
 }: InputRootProps) {
-  // @ts-expect-error - "aria-label" is missing from HTMLInputAttributes
+  // TODO Remove next line once updated to preact 10.13.0
+  // @ts-ignore-next - "aria-label" is missing from HTMLInputAttributes.
```

Comment:
Preact 10.13 adds the missing `aria-label`. The `@ts-expect-error` annotation throws if TS does not find an error, so I have replaced it with `@ts-ignore-next`.

Once updated to preact 10.13, I will create another PR removing this altogether.